data/brands/amenity/recycling.json

```
@@ -107,6 +107,20 @@
         "name": "Ecopunto"
       }
     },
+    {
+      "displayName": "Islamic Relief",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "amenity": "recycling",
+        "brand": "Islamic Relief",
+        "brand:wikidata": "Q239714",
```

Looks like this recycling amenity have a wikipedia
https://en.wikipedia.org/wiki/Islamic_Relief
I think we can add it.
```suggestion
        "brand:wikidata": "Q239714",
        "brand:wikipedia":"en:Islamic Relief",
```